### PR TITLE
add vpa to dashboard metrics scraper

### PR DESCRIPTION
**Pull request by demonCoder95**
Kubernetes dashboard metrics scraper pod restarted because of OOMKill. This was due to scaling as part of the GLT today. There is a pattern of these restarts due to load test scaling.

This PR adds a VPA to the metrics-scraper for Kubernetes dashboard to avoid this.
Screenshot 2023-10-18 at 13 03 21

### CODE REVIEW

File: cluster/manifests/dashboard/scraper-vpa.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: scraper-vpa
```
Comment (mikkeloscar) [inline cluster/manifests/dashboard/scraper-vpa.yaml]: nit: let's call if `dashboard-metrics-scraper-vpa`

Comment (demonCoder95) [reply cluster/manifests/dashboard/scraper-vpa.yaml]: Done in the latest commit.

**Comment by linki** [comment]
👍

**Comment by demonCoder95** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Comment by demonCoder95** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**